website/source/docs/agent/options.html.md

```diff
@@ -757,6 +757,16 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
   <br><br>
   The following sub-keys are available:
 
+  * <a name="disable_endpoints"></a><a href="#disable_endpoints">`disable_endpoints`</a>
```

Comment:
This would also disable any consul commands that use the underlying api, right? Is it worth calling out here?